.github/actions/make-in-docker/Dockerfile

```diff
@@ -1,21 +1,28 @@
 # vim: set expandtab sw=4
 FROM golang:1.12-buster
 
+RUN chmod a+rx $HOME && useradd --uid 9999 --user-group --create-home browser
+
 # Sort the package names!
+# firefox-esr: provides deps for Firefox (we don't use ESR directly)
 # openjdk-11-jdk: provides JDK/JRE to Selenium & gcloud SDK
 # python-crcmod: native module to speed up CRC checksum in gsutil
-# sudo: used in Makefile (no-op inside Docker)
 RUN apt-get update -qqy && apt-get install -qqy --no-install-suggests \
         curl \
         equivs \
+        firefox-esr \
         lsb-release \
         openjdk-11-jdk \
         python-crcmod \
         python3.7 \
         sudo \
         tox \
         wget \
-        xvfb
+        xvfb && \
+    rm /usr/bin/firefox
+
+# Overwrite sudoers to discard the secure_path setting to make go available.
```

Comment:
Sorry, bit of term overload here with `make` and `go` :D. Does this have anything to do with Makefiles, or is it just that `go` won't work when `secure_path` is set? (Which doesn't work because...?)

Reply:
It's because the official Go Docker image (the base image of ours) installs Go to /usr/local/go and appends `/usr/local/go/bin` to `PATH`. Debian by default resets `PATH` to `secure_path`, which is hard coded to a list of common paths like `/bin:/usr/bin`, inside `sudo`, to prevent accidentally running a non-system binary as root.